apps/sms/views/inbox/style/inbox.css

```diff
@@ -106,9 +106,13 @@
 }
 
 [data-type="list"] .unread aside.icon-unread {
-  background: url("images/unread.png") no-repeat left center / 1rem;
+  background-image: url("images/unread.png");
+  background-repeat: no-repeat;
```

Comment:
I think you can keep the background property for the image and repeat values.
rationale: it sets all other properties to their initial values (so we don't risk being bitten by some inherited stuff here)